.github/workflows/python-tests.yml

```
@@ -21,16 +21,43 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     
+    - name: Create data directories
+      run: |
+        mkdir -p data/contexts/default
+        mkdir -p data/contexts/_context_history
+        mkdir -p data/sessions
+        mkdir -p data/llmops
+        mkdir -p data/temp
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)